src/egon/data/datasets/scenario_capacities.py

```diff
@@ -658,15 +658,15 @@ def eGon100_capacities():
     cwd = Path(".")
 
     if execute_pypsa_eur_sec:
```

Comment:
The value for this variable is set manually some lines above to False. In that way, this first part of the if statement will never run. Additionally, the variable's name can be misleading since we no longer use pypsa-eur-sec.